lib/sigsci-agent.sh

```diff
@@ -122,7 +122,8 @@ then
     fi
     else
       # download the build pack from a local repo
-      curl -s --retry 45 --retry-delay 2 -o sigsci-agent_${SIGSCI_AGENT_VERSION}.tar.gz "${SIGSCI_AGENT_DOWNLOAD_URL}/sigsci-agent_${SIGSCI_AGENT_VERSION}.tar.gz" > /dev/null
+      echo "-----> Using local SigSci Agent repository ${SIGSCI_AGENT_DOWNLOAD_URL}"
+      curl -s --retry 45 --retry-delay 2 -o sigsci-agent_${SIGSCI_AGENT_VERSION}.tar.gz $SIGSCI_AGENT_DOWNLOAD_URL > /dev/null
```

Comment:
SIGSCI_AGENT_VERSION isn't guaranteed to be set here; you could update the download file to just be `sigsci-agent.tar.gz` in both branches of the if and unpack without the version in the filename.

Reply:
removed the version from downloaded file name